packages/cli/src/databases/entities/AbstractEntity.ts

```diff
@@ -44,7 +43,7 @@ function mixinStringId<T extends Constructor<{}>>(base: T) {
 	return Derived;
 }
 
-function mixinTimestamps<T extends Constructor<{}>>(base: T) {
+function mixinTimestamps<T extends Class<{}, any[]>>(base: T) {
```

Comment:
```suggestion
function mixinTimestamps<T extends Class<object>(base: T) {
```

Reply:
unfortunately this does not work. TS is very picky about class mixins 
![image](https://github.com/n8n-io/n8n/assets/196144/0b85067d-76ce-4f1f-bb90-6d20bdbb140c)